### ISSUE TITLE
Example for OHLC information does not work

### ISSUE THREAD

**Issue by mccoydj1** [bmoscon/cryptofeed#341]
The example code here
https://github.com/bmoscon/cryptofeed/blob/master/examples/demo_ohlcv.py

fails with this error
TypeError: __call__() got an unexpected keyword argument 'order_type'

**Comment by bmoscon** [comment]
I just ran it, no issues

```
python examples/demo_ohlcv.py
{'BTC-USD': {'open': Decimal('18430.01'), 'high': Decimal('18449.2'), 'low': Decimal('18430'), 'close': Decimal('18432.34'), 'volume': Decimal('56.11503510'), 'vwap': Decimal('18436.72860436931277977584300')}, 'ETH-USD': {'open': Decimal('586.29'), 'high': Decimal('589.77'), 'low': Decimal('586.2'), 'close': Decimal('588.72'), 'volume': Decimal('1395.72490024'), 'vwap': Decimal('588.5484344902925896946667488')}, 'BCH-USD': {'open': Decimal('290.83'), 'high': Decimal('291.27'), 'low': Decimal('289.9'), 'close': Decimal('290.43'), 'volume': Decimal('421.31182434'), 'vwap': Decimal('290.4526434126368151483531183')}}
```

What version of Python? What version of the code? Are you using latest from githuh, or what is in pypi?

**Comment by bmoscon** [comment]
I'm pretty sure you are just running with outdated code, this was fixed a few days ago in 1.6.2, it hasnt been released to pypi yet, so use the latest code from github or wait for the next release

**Comment by mccoydj1** [comment]
oops sorry!
